tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -198,24 +202,63 @@ Machine K3S config is the final piece needed for the config to run K3S provision
 ```
 
 ## Custom Cluster
-For custom clusters, the below config is needed, only AWS/EC2 will work.
-**Ensure you have nodeProviders in provisioningInput**
+For custom clusters, no machineConfig or credentials are needed. Currently only supported for ec2.
 
+Dependencies:
+* **Ensure you have nodeProviders in provisioningInput**
+* ensure len(awsEC2Config) == len(nodesAndRoles)
```

Comment:
I think it's more clear if you say don't have the code in the readme. For example, `Make sure that the defined length of `awsEC2Config` is equal to the defined lenght of `nodesAndRoles`.